docs/ProductPrice.md

```diff
@@ -1,6 +1,8 @@
 📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion). 
 
-# Product Price
+# Product Price ![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red)
+
+:warning: **The Product Price block has been deprecated in favor of the [Product Price](https://vtex.io/docs/components/all/vtex.product-price@1.3.0/) app**. Although support for this block is still granted, we strongly recommend you to update your store theme with the Product Price's blocks in order to keep up with the component's evolution.
```

Comment:
Better not to fix the app version in the documentation URL :)

```suggestion
:warning: **The Product Price block has been deprecated in favor of the [Product Price](https://vtex.io/docs/components/all/vtex.product-price/) app**. Although support for this block is still granted, we strongly recommend you to update your store theme with the Product Price's blocks in order to keep up with the component's evolution.
```

Reply:
> Better not to fix the app version in the documentation URL :)

Ué